### PR TITLE
document entry level opts

### PR DESCRIPTION
**Pull request by yoshuawuyts**
A few people pointed out that some of the docs weren't as clear as they could be. Hopefully this helps a little :sparkles:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,29 +270,32 @@ Whenever a change in the internal graph occurs.
...
+
+### `compiler.documents(routename, [opts], done(err, { buffer }))`
 Output an HTML bundle for a route. Routes are determined based on the project's
 router. Pass `'/'` to get the default route.
 
 - __opts.state:__ Will be passed the render function for the route, and inlined
   in the `<head>` of the body as `window.initialState`.
 
-### `compiler.scripts(filename, done(err, buffer))`
+### `compiler.scripts(filename, done(err, { buffer }))`
```
Comment (goto-bus-stop) [inline README.md]: should we do `{ buffer, hash }` then here?

File: README.md [hunk 2]
```diff
@@ -270,29 +270,32 @@ Whenever a change in the internal graph occurs.
 
 ## API
 ### `compiler = bankai(entry, [opts])`
-Create a new bankai instance. Takes either an entry file location, or an array
-of files.
+Create a new bankai instance from the route to JavaScript file. The following
```
Comment (goto-bus-stop) [inline README.md]: > Create a new bankai instance from the route to JavaScript file

I'm not sure how to parse this sentence 🙈

File: README.md [hunk 3]
```diff
@@ -270,29 +270,32 @@ Whenever a change in the internal graph occurs.
...
+options are available:
 
-### `compiler.documents(routename, [opts], done(err, buffer))`
+- __opts.quiet:__ Defaults to `false`. Don't output any data to `stdout`. Useful
```
Comment (goto-bus-stop) [inline README.md]: can we add `opts.watch` here too?

Comment (yoshuawuyts) [reply README.md]: Oh, is that an option? Yeah, we probably should then. Thought it wasn't tho :eyes:

File: README.md (hunk 2 above)
Comment (yoshuawuyts) [reply README.md]: lol, yeah fair

Let's redo it as:
> Create a new bankai instance. Takes a path to a JavaScript file as the first argument. (...)

Comment (goto-bus-stop) [reply README.md]: 👌

**Pull request merged**